src/Controller/AdminController.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use App\Entity\User;
+use App\Form\AddArticleType;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Response;
+
+class AdminController extends AbstractController
+{
+    /**
+     * @Route("/admin/edit-user", name="edit-user")
+     */
+    public function editUsersAction()
+    {
+        $repository = $this->getDoctrine()->getRepository(User::class);
+
+        $users = $repository->findAll();
+
+        return $this->render('admin/editUsers.html.twig', [
+            'users' => $users,
+        ]);
+    }
+
+
+
+    /**
+     * @param Request $request
+     * @param User $id
+     * @Route("/admin/edit-user/delete/{id}", name="delete")
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
+     */
+    public function deleteUserAction(Request $request, User $id)
+    {
+
+            $em = $this->getDoctrine()->getManager();
+            $product = $em->getRepository(User::class)->find($id);
```

Comment:
you already have a User entity into $id variable. No reason to make another request to database. Also why is it called $product?